tests/session/session-debug-004.phpt

```diff
@@ -34,5 +34,11 @@ object(MongoDB\Driver\Session)#%d (%d) {
   NULL
   ["server"]=>
   NULL
+  ["inTransaction"]=>
+  NULL
+  ["transactionState"]=>
+  NULL
```

Comment:
I forgot that `Session::endSession` actually sets `intern->client_session` to null, so this is a good test that you added `if (client-session) {` around your new fields.